ironfish/src/memPool/feeEstimator.ts

```diff
@@ -226,7 +226,7 @@ export class FeeEstimator {
   private async getPendingTransactionSize(
     sender: Account,
     receives: { publicAddress: string; amount: bigint; memo: string }[],
-    estimateFeeRate?: bigint,
+    feeRate: bigint,
   ): Promise<number> {
     let size = 0
     size += 8 // spends length
```

Comment:
nit: can you use a constant like Numeric_Value_Length to replace the hardcoded number. and new constants for signature and expiration length too. a good place for these constants are in ironfish/src/primitives/transaction.ts

Reply:
That's a good idea! I'll create a separate issue for this; it's tangential to the scope of these changes